### ISSUE TITLE
[BUG] RFT changes attributes that's in __all__

### ISSUE THREAD

**Issue by EmberLightVFX** [dashingsoft/pyarmor#1598]
I'm building a module that I wish to obfuscate using RFT.
The docs says:
```
What does RFT mode not change?
    * all the strings defined in the module attribute __all__
``` 

but from my tests it seems like RFT still changes the names that's in under __all__.

Here is a very simpel test-script:
[src.zip](https://github.com/dashingsoft/pyarmor/files/13378439/src.zip)
It has two files in folder called src: `__init__.py` and `main.py`
`__init__.py` contains
```python
from .main import printTest

__all__ = ["printTest"]
```
and `main.py` contains
```python
def printTest():
    print("hello")
```

I then try to obfuscate the package using `pyarmor gen -r -i --enable-rft src` and then import and run it I'm getting
`cannot import name 'pyarmor__1' from 'dist.src.main'`

Checking the RFT trace-files I can see that the function printTest has bin renamed even tho it's added to `__all__` in the `__init__` file.

Is this a bug or do I just don't understand what `all the strings defined in the module attribute __all__` means?

```
INFO     Python 3.10.11
INFO     Pyarmor 8.4.3 (group), 006141, aesl
INFO     Platform windows.x86_64
```

**Comment by jondy** [comment]
This line should be in the `main.py`
```python
__all__ = ["printTest"] 
```

Or use rft option `exclude`

**Comment by EmberLightVFX** [comment]
`__all__` isn't used inside a normal python file. Maybe in a single file python script for some reason but 99% of the time you have an `__init__.py` file containing the `__all__` variable. This seems like a design flaw with pyarmor?

As I'm packaging a whole folder with recursive enabled, shouldn't pyarmor check all files for the __all__ tag and add them to the exclude?

Currently I have to manually go through all my `__init__.py` files and add their content to the rft's exclude. As the module evolve doing this each and every time to make sure I don't miss anything that might have changed or gotten added takes a long time.

**Comment by jondy** [comment]
Got it. I'll try to add this feature to keep imported names in `__all__`

**Comment by EmberLightVFX** [comment]
Works as expected now :) Thanks for the fix!

**Comment by jondy** [comment]
Fixed in v8.4.4
